components/style/stylesheets.rs

```diff
@@ -1836,9 +1854,10 @@ impl<'a, 'b> QualifiedRuleParser for NestedRuleParser<'a, 'b> {
     type QualifiedRule = CssRule;
 
     fn parse_prelude(&mut self, input: &mut Parser) -> Result<SelectorList<SelectorImpl>, ()> {
+        let ns = self.context.namespaces.expect("namespaces must be set when parsing rules").read();
```

Comment:
It'd be nice not to have atomic operations every time we parse selectors... Please file an issue on it?